showcase/src/app/demo/collapsible/collapsible.component.html

```diff
@@ -0,0 +1,80 @@
+<example-viewer [extensions]="['HTML', 'CSS', 'TS']" filesPath="collapsible/collapsible-default" exampleTitle="SHOWCASE.DEMO.COLLAPSIBLE.DEFAULT_ICON">
+	<stark-collapsible 
+			htmlId="id0" 
```

Comment:
Let's provide more meaningful IDs to our variables/components to prevent developers from developing those bad practices.